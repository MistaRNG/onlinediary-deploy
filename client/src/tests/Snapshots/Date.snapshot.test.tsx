import React from 'react';
import { render } from '@testing-library/react';
import LongDateDisplay from '../../features/journal/components/Editor/Date';
import '@testing-library/jest-dom';
import { describe, it, expect } from '@jest/globals';

describe('LongDateDisplay Component Snapshot Tests', () => {
  it('should render the LongDateDisplay component correctly', () => {
    // ARRANGE & ACT
    const { asFragment } = render(<LongDateDisplay date={new Date('2024-09-06')} />);

    // ASSERT
    expect(asFragment()).toMatchSnapshot();
  });
});
