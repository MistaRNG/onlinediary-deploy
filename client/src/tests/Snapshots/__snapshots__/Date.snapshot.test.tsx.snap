// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LongDateDisplay Component Snapshot Tests should render the LongDateDisplay component correctly 1`] = `
<DocumentFragment>
  <div>
    Friday, September 6, 2024
  </div>
</DocumentFragment>
`;
